integration/spark/src/main/scala/org/apache/spark/sql/execution/command/mutation/CarbonProjectForUpdateCommand.scala

```diff
@@ -158,8 +159,6 @@ private[sql] case class CarbonProjectForUpdateCommand(
                 "for the update key")
             }
           }
-          // handle the clean up of IUD.
-          CarbonUpdateUtil.cleanUpDeltaFiles(carbonTable, false)
```

Comment:
if we remove it, we will not support the legacy store.
we need to check it at first(whether is ok or not if we not support the legacy store).

Reply:
@QiangCai , @marchpure : In the release notes or upgrade guide, we can suggest to the user that if update files are present, compact (horizontal) before upgrading. Just my opinion.

Reply:
Clean Files Command maybe a better solution.

Reply:
@ajantha-bhat ok